providers/song-info.js

```diff
@@ -95,15 +96,15 @@ const registerProvider = (win) => {
 		await handleData(responseText, win);
 		handlingData = false;
 		callbacks.forEach((c) => {
-			c(songInfo);
+			c(songInfo, "video-src-changed");
```

Comment:
Unused right now, but now a plugin can now whats the reason for the songInfo callback, if relevant